observation_portal/sciapplications/templates/sciapplications/ddt_submitted.txt

```diff
@@ -8,8 +8,7 @@ You will be contacted in due course once we have reached a decision.
 
 You may view your proposal here:
 
-https://observe.lco.global{{ ddt.get_absolute_url }}
+{{ detail_url }}
 
 Warm Regards,
-Lisa Storrie-Lombardi
```

Comment:
I figure just the organization name of LCO or LCO.global is fine here, rather than coding in a Directors name...

Reply:
I like the idea of {{org_name}} director - then we can leave the warm regards part and it sounds more personal...